doc/user/radial_and_axial_expansion.rst

```diff
@@ -60,27 +57,188 @@ length, :math:`L_0`,
+:nuref:`hot_density` illustrate how the radius, height, volume, density, and mass are updated for a
+component during thermal expansion.
+
+.. list-table:: Example Calculation of Radial and Axial Thermal Expansion for a Cylindrical Component
+   :widths: 33 33 33
+   :header-rows: 1
+
+   * - Component Temperature
+     - 20°C
+     - 1000°C
+   * - Radius
+     - 0.25 cm
+     - 0.251 cm
+   * - Height
+     - 5.0 cm
+     - 5.01 cm
+   * - Volume
+     - 0.982 :math:`cm^3`
+     -  0.988 :math:`cm^3`
+   * - Density
+     - 1.0 g/cc
+     - 0.994 g/cc
+   * - Mass
+     - 0.982 g
+     - 0.982 g
+
+.. math::
+   :name: hot_radius
+
+    r(T_h) = 0.25 \left(1 + \left(2\times 10^{-6}(1000 − 20)\right) = 0.251 cm
+
+.. math::
+   :name: hot_height
+
+    h(T_h) = 5.0 \left(1 + (2\times 10^{-6}(1000 − 20)\right) = 5.01 cm
+
+.. math::
+   :name: hot_volume
+   
+    V(T_h) = \pi (0.251)^2 5.01 = 0.988 cm^3
+
+.. math::
+   :name: hot_density
+
+    \rho(T_h) = \frac{1.0}{\left(1 + 2\times 10^{-6}(1000 − 20)\right)} = 0.994 \frac{g}{cc}
+
+.. math::
+   :name: hot_mass
+
+    m(T_h) = 0.994 \times 0.988  = 0.982 g
+
+When two or more components exist within the Block, the overall height change of the Block is driven
+by an axial expansion "target Component" (e.g., fuel). This target is selected (either automatically
+based on component flags or manually in the blueprints) to ensure that specific component has its
+mass conserved post-expansion. Note, the remaining components also have their mass conserved on the
+assembly-level; however intra-block mass conservation is lost as there is a redistribution of mass
+between blocks for non-target components.
+
+Radial thermal expansion occurs for each component in a given block. Mechanical contact between
+components is not accounted for, meaning that the radial expansion of one Component is independent
+from the radial expansion of the others. Solid components may be linked to gas/fluid components
+(i.e., sodium bond, helium) and the gas/fluid area is allowed to expand and contract with changes in
+Component temperatures. Alternatively, the axial thermal expansion of components within a Block do
+influence the positions of components in neighboring blocks for a given Assembly. So, while axial
+thermal expansion evaluations of each Block are treated independently, the axial mesh points are
+updated to account for the physical material displacements across the entire assembly length.
+
+The following two tables provide illustrations of the axial thermal expansion process for an example
+core assembly. In this example there are four main block types defined: Shield, Fuel, Plenum, and
+Dummy.
+
+.. figure:: /.static/axial_expansion_components.png
+
+    Illustration of Components for Axial Thermal Expansion Process
+
+.. figure:: /.static/axial_expansion_process.png
+
+    Simplified Illustration of Axial Thermal Expansion Process for a Core Assembly
```

Comment:
more odd scaling here for these figures. The first is 'yuge'. The second is ok, I think.

Reply:
try now